Automattic/vip-go-mu-plugins
# updates query-monitor to 3.6.4

## Description

Upgrades to Query Monitor 3.6.1

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

* Deploy
* Check that query Monitor tool works as expected

![image](https://user-images.githubusercontent.com/19240162/94155583-6494da80-fe7f-11ea-95da-7552bc27eaac.png)
